Makefile

```diff
@@ -93,11 +106,23 @@ logs: ## display app logs (follow mode)
 	@$(COMPOSE) logs -f app
 .PHONY: logs
 
-run: ## starts supported backends servers
```

Comment:
There are still some references for this command in the Makefile:

At line 65 we have `es-index: run`
At line 136 we have `test: run`

Reply:
Good catch! :pray: